src/main/java/org/isf/lab/gui/LabNew.java

```diff
@@ -850,7 +851,7 @@ public void actionPerformed(ActionEvent e) {
 					
 					int selectedRow = jTableExams.getSelectedRow();
 					if (selectedRow < 0) {
-						MessageDialog.error(LabNew.this,"angal.labnew.pleaseselectanexam");
+						MessageDialog.error(LabNew.this,"angal.lab.pleaseselectanexam.msg");
```

Comment:
should we keep only the `angal.lab.` root?

Reply:
sure; but I was trying not to make resource bundle changes that might force Transifex to run.

Reply:
I see